analysis/11.analyze.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    GENE_SEP = ";"
</pre><p>I'm not understanding how these last 3 would differ based on dataset. To me it seems they could be constants. Let's discuss in person!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheeseman-lab/brieflow-analysis/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>